src/runtime/runtime.js

```diff
@@ -349,7 +349,7 @@ export class Runtime {
   saveSubscription(saveSubscriptionRequestCallback) {
     return this.configured_(true)
         .then(runtime => {
-          runtime.saveSubscription(saveSubscriptionRequestCallback);
+          return runtime.saveSubscription(saveSubscriptionRequestCallback);
```

Comment:
There should be a test for this.

Reply:
Dima can you clarify? there are tests that verify delegation to configured run time to call the right flow. I just updated the tests. can you please take a look and let me know if you still think some tests are missing?

Reply:
Done